### ISSUE TITLE
Error: Cannot find module '@tryghost/mg-mediascraper'

### ISSUE THREAD

**Issue by phrasenmaeher** [TryGhost/migrate#530]
I am using your tool to migrate from other sources to Ghost. However, it throws the error mentioned in the title.

Full error stack is below:
```(base) pascal@MacBook-Pro ghost % migrate
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@tryghost/mg-mediascraper'
Require stack:
- /opt/homebrew/lib/node_modules/@tryghost/migrate/sources/ghost.js
- /opt/homebrew/lib/node_modules/@tryghost/migrate/commands/ghost.js
- /opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/sywac/api.js
- /opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/@tryghost/pretty-cli/lib/pretty-cli.js
- /opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/@tryghost/pretty-cli/index.js
- /opt/homebrew/lib/node_modules/@tryghost/migrate/bin/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/@tryghost/migrate/sources/ghost.js:4:24)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/@tryghost/migrate/commands/ghost.js:1:15)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/sources/ghost.js',
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/commands/ghost.js',
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/sywac/api.js',
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/@tryghost/pretty-cli/lib/pretty-cli.js',
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/node_modules/@tryghost/pretty-cli/index.js',
    '/opt/homebrew/lib/node_modules/@tryghost/migrate/bin/cli.js'
  ]
}

Node.js v17.6.0
```

**Comment by jdibiccari** [comment]
I am also encountering this error. Any guidance?

**Comment by PaulAdamDavis** [comment]
Thanks for reporting! This has been fixed in [0.23.2](https://github.com/TryGhost/migrate/releases/tag/%40tryghost%2Fmigrate%400.23.2)
